app/models/mentor.rb

```diff
@@ -0,0 +1,26 @@
+class Mentor < ApplicationRecord
+  include Mentors::AllowlistMentors
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :jwt_authenticatable, jwt_revocation_strategy: self
+
+  has_many :allowlisted_mentors_jwts, dependent: :destroy
+  has_many :bookings, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :mentor_technologies, dependent: :destroy
+  has_one_attached :avatar
+
```

Comment:
- It looks like not all the models have required validations, are you sure you couldn't improve by adding more validations for these models? If the answer is definitively not, please ignore this comment. 🐶